components/mailgun/sources/new-log-data/new-log-data.js

```diff
@@ -0,0 +1,81 @@
+const common = require("../common-webhook");
```

Comment:
Since this is a timer-based source, do you want to require everything from `common-webhook`?